recipe/meta.yaml

```diff
@@ -46,12 +45,92 @@ test:
     - PyHyperScattering
   commands:
     - pip check
-
+outputs:
+  - name: pyhyperscattering-base
+    build:
+      script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+    run_exports:
+      # keep downstream ABI-compatible on minor bumps
+      - {{ pin_subpackage("pyhyperscattering-base", max_pin="x.x") }}
+  - name: pyhyperscattering
+    build:
+      noarch: python
+      script: "{{ PYTHON }} -c 'import sys; sys.exit(0)'"
+    requirements:
+      host:
+        - python {{ python_min }}
+      run:
+        - python >={{ python_min }}
+        - {{ pin_subpackage("pyhyperscattering-base", max_pin="x.x.x") }}
+        # ---------- all-nogpu extras ----------
+        - tiled[client] >=0.1.0a74
+        - bluesky-tiled-plugins
+        - pyopencl
+        - dask
+        - holoviews
+        - hvplot
+        - sphinx
+        - pydata-sphinx-theme
+        - pytest
+        - black
+        - codecov
+        - pylint
+        - flake8
+        - coverage
+        - pybind11
+        - wheel
```

Comment:
These all look like development dependencies. Is this intended?

Reply:
Yes; this is the approach I'm trying to roll out here for GPU.  I think in light of this and to minimize further complexity I'll remove the development dependencies.